lib/vm/src/libcalls.rs

```diff
@@ -35,13 +35,17 @@
 //!   }
 //!   ```
 
+#![allow(missing_docs)] // For some reason lint fails saying that `LibCall` is not documented, when it actually is
```

Comment:
If I had to guess, it's one of the derive macros. Derive macros can edit code streams... I don't know if they can add variants to an enum (probably not) but they can definitely make new `impl YourType { fn my_function() {  ... } }` blocks which could be failing the doc thing.

I think we shouldn't be asserting docs statically, but as part of a CI lint step in any case but until then, the narrower the `allow` directive is, the better.

Reply:
I'll keep it as it is now as I don't have any other way of solving it for now :(